### ISSUE TITLE
adding dependency libs onto the project is not a good practice

### ISSUE THREAD

**Issue by BogdanLivadariu** [priyanshus/mobile-game-automated-tests#1]
why don't you make it a maven project and specify the dependencies inside the `pom.xml` file this way, you should not take care of adding dependencies inside the source code of the project

**Comment by priyanshus** [comment]
yes, I agree. Even I do not like to have dependencies in the project itself. It is just a proof of concept so I did not enhanced it much.

But for sure, I will be doing that in couple of weeks.
